test/relay/relay.node.ts

```diff
@@ -192,8 +198,14 @@ describe('Dialing (via relay, TCP)', () => {
     const dialAddr = relayAddr.encapsulate(`/p2p/${relayLibp2p.peerId.toString()}`)
 
     const connection = await srcLibp2p.dial(dialAddr)
-    const stream = await connection.newStream('/libp2p/circuit/relay/0.1.0')
-
+    /* eslint-disable-next-line no-console */
+    console.log('>>>>>> connection established')
```

Comment:
NOTE: If the connection is established, `connection.newStream` fails because of a reset.